python/pyspark/sql/connect/client.py

```diff
@@ -42,6 +43,125 @@
 logging.basicConfig(level=logging.INFO)
 
 
+class ChannelBuilder:
+    """
+    This is a helper class that is used to create a GRPC channel based on the given
+    connection string per the documentation of Spark Connect.
+    """
+
+    PARAM_USE_SSL = "use_ssl"
+    PARAM_TOKEN = "token"
+    PARAM_USER_ID = "user_id"
+
+    DEFAULT_PORT = 15002
```

Comment:
BTW, there is no equivalent Spark Conf, etc. for Python?

Reply:
Unfortunately not.